packages/desktop-client/src/components/manager/BudgetList.jsx

```diff
@@ -294,47 +298,51 @@ export function BudgetList() {
         >
           <RefreshButton
             onRefresh={() => {
-              getUserData();
-              loadAllFiles();
+              dispatch(getUserData());
+              dispatch(loadAllFiles());
             }}
           />
         </View>
       </View>
       <BudgetTable
         files={files}
-        actions={actions}
         onSelect={file => {
           if (file.state === 'remote') {
-            downloadBudget(file.cloudFileId);
+            dispatch(downloadBudget(file.cloudFileId));
           } else {
-            loadBudget(file.id);
+            dispatch(loadBudget(file.id, `Loading ${file.name}...`));
           }
         }}
-        onDelete={file => pushModal('delete-budget', { file })}
+        onDelete={file => dispatch(pushModal('delete-budget', { file }))}
       />
       <View
         style={{
           flexDirection: 'row',
           justifyContent: 'flex-end',
           padding: 25,
-          paddingLeft: 5,
+          gap: 15,
         }}
       >
         <Button
           type="bare"
           style={{
-            marginLeft: 10,
+            ...narrowButtonStyle,
             color: theme.pageTextLight,
           }}
-          onClick={e => {
-            e.preventDefault();
-            pushModal('import');
+          onClick={() => {
+            dispatch(pushModal('import'));
           }}
         >
           Import file
         </Button>
 
-        <Button type="primary" onClick={onCreate} style={{ marginLeft: 15 }}>
+        <Button
+          type="primary"
+          onClick={onCreate}
+          style={{
+            ...narrowButtonStyle,
+          }}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
          style={narrowButtonStyle}
```